config/sync/core.entity_view_display.paragraph.os2loop_section_page_views_refer.default.yml

```diff
@@ -3,26 +3,24 @@ langcode: en
 status: true
 dependencies:
   config:
-    - field.field.paragraph.os2loop_section_page_views_refer.os2loop_section_page_view
+    - field.field.paragraph.os2loop_section_page_views_refer.field_os2loop_section_page_block
     - field.field.paragraph.os2loop_section_page_views_refer.os2loop_section_page_view_header
     - field.field.paragraph.os2loop_section_page_views_refer.os2loop_section_page_view_text
     - paragraphs.paragraphs_type.os2loop_section_page_views_refer
   module:
+    - block_field
     - text
-    - viewsreference
 id: paragraph.os2loop_section_page_views_refer.default
 targetEntityType: paragraph
 bundle: os2loop_section_page_views_refer
 mode: default
 content:
-  os2loop_section_page_view:
-    weight: 2
+  field_os2loop_section_page_block:
```

Comment:
Remove the `field_` prefix (cf. https://github.com/itk-dev/os2loop/tree/develop/docs/development#fields).

Reply:
Changed